recipes/gnupg-py/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gnupg" %}
+{% set version = "2.3.1" %}
+{% set sha256 = "8db5a05c369dbc231dab4c98515ce828f2dffdc14f1534441a6c59b71c6d2031" %}
+
+package:
+  name: {{ name|lower }}-py
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - typestubs = pytypes.stubfile_2_converter:main
```

Comment:
Where is this coming from?

Reply:
remnant from another one i did,  removing